node/cn/handler.go

```diff
@@ -461,7 +461,7 @@ func (pm *ProtocolManager) processConsensusMsg(msgCh <-chan p2p.Msg, p Peer, add
 			_, err := handler.HandleMsg(addr, msg)
 			// if msg is istanbul msg, handled is true and err is nil if handle msg is successful.
 			if err != nil {
-				p.GetP2PPeer().Log().Error("ProtocolManager failed to handle consensus message", "msg", msg, "err", err)
+				p.GetP2PPeer().Log().Warn("ProtocolManager failed to handle consensus message. This can happen during block synchronization", "msg", msg, "err", err)
```

Comment:
How about checking specific error as you mentioned?
All errors happen caused by that case?

Reply:
@KimKyungup Yes. I checked HandleMsg() and related codes, the only error which can be returned was "ErrStoppedEngine". And the engine could be stopped only by the downloader.

I think the original author might assumed more cases, but no other error is present at the moment in our code base.

Reply:
@KimKyungup FYI, https://groundx.atlassian.net/jira/software/projects/TB/boards/36/backlog?assignee=5ab9f16ca142832a9381c64a&selectedIssue=TPT-597

Reply:
Thanks. @andybclee  :) I understand.